packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -198,9 +226,45 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                 }
             },
             // eth addresses are valid without checksum but Trezor displays them as checksummed
-            checksum: (value: string) => {
-                if (networkType === 'ethereum' && !checkAddressCheckSum(value)) {
-                    return translationString('RECIPIENT_IS_NOT_VALID');
+            checksum: async (address: string) => {
+                if (networkType === 'ethereum') {
+                    const updatedValues = addValue(address); // Get updated values immediately
+                    const params = {
+                        descriptor: address,
+                        coin: symbol,
+                    };
+
+                    //they are not the same but the second one is checksum of the first one, in this case show the info that the address was automatically checksummed
+                    if (
+                        updatedValues &&
+                        updatedValues[0] &&
+                        updatedValues[1] &&
+                        updatedValues[0] !== updatedValues[1] &&
+                        toChecksumAddress(updatedValues[0]) === updatedValues[1]
+                    ) {
+                        setAddressChecksumInfo({
+                            text: 'Address converted to checksum format.',
+                        });
+                    } else {
+                        setAddressChecksumInfo(undefined);
+                    }
+
+                    //1.If the address is used but unchecksummed, then Suite will automatically convert the address to the correct checksummed form and inform the user as described in the OP.
+                    const result = await TrezorConnect.getAccountInfo(params); //TODO handle error
+                    if (result.success && result.payload.history.total !== 0) {
+                        if (!checkAddressCheckSum(address)) {
+                            setValue(inputName, toChecksumAddress(address), {
+                                shouldValidate: true,
+                            });
+                        }
+                    }
+                    //2. if address is not checksummed at all and not found in blockbook
+                    //offer to checksum width a button
+                    if (result.success && result.payload.history.total == 0) {
+                        if (address == address.toLowerCase()) {
```

Comment:
use `===` + you can wrap it into ` if (result.success) {`
```suggestion
                    if (result.success) {
                        if (result.payload.history.total !== 0 && !checkAddressCheckSum(address)) {
                            setValue(inputName, toChecksumAddress(address), {
                                shouldValidate: true,
                            });
                        }
                  
                        if (result.payload.history.total === 0 && address === address.toLowerCase()) {
```